### ISSUE TITLE
Cura 5 extrusion errors

### ISSUE THREAD

**Issue by GT-Geo** [Ultimaker/Cura#12104]
### Application Version

5

### Platform

Mac 12.3

### Printer

Ender 3 V2

### Reproduction steps

Sliced a file, portions of the text area have errors. 

The first screen shot is Cura V 4.13.1, the second screen shot is Cura 5. Same settings for both version
<img width="1330" alt="Cura 4 13 1" src="https://user-images.githubusercontent.com/104281051/167013250-f13f3fd6-ba3d-445d-a8d5-4eba4e6fee8c.png">
<img width="1315" alt="Cura 5" src="https://user-images.githubusercontent.com/104281051/167013259-88e979cd-2576-4d29-8e16-7f3e68c73b54.png">
.

### Actual results

Portions of the text area have errors.

### Expected results

Expect clean results like 4.13.1

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

[HH Bust.stl.zip](https://github.com/Ultimaker/Cura/files/8635520/HH.Bust.stl.zip)

**Comment by GregValiant** [comment]
When the warning comes up that a model isn't "watertight" that means it has open seams someplace.  That often causes errors in the slice because Cura cannot tell what is the inside and what is the outside.
After having the model repaired I didn't get any problems slicing it.

Here is the report from [formware.](https://formware.co/OnlineStlRepair)
--> 0 Naked edges (?)
--> 0 Planar holes (?)
--> 0 Non-planar holes (?)
--> 17 Non-manifold edges (?)
--> 20 Inverted faces (?)
--> 8 Degenerate faces (?)
--> 0 Duplicate faces (?)
--> 0 Disjoint shells (?)
-> Repairing: 100.00%
----- Repair completed in 10255ms ------
-> Vertex count changed from 32411 to 32390 (-21)
-> Triangle count changed from 64854 to 64806 (-48)

My guess is that when the text was added to the model that the letters weren't imbedded in the model surface and that may have left gaps.

**Comment by GT-Geo** [comment]
Thanks for your input, however I tried to repair the file using formware and re-slice it in Cura 5 with the same results
<img width="1001" alt="Screen Shot 2022-05-05 at 8 15 16 PM" src="https://user-images.githubusercontent.com/104281051/167046823-563ec6b3-e5c9-404e-9b7b-582598e45afc.png">
.

**Comment by GregValiant** [comment]
With the model loaded and your settings how you like - use the "File | Save Project" command.  That will create a 3mf file with you printer the model and your settings.  Zip the file and post it here.

**Comment by GT-Geo** [comment]
Here's the 3mf file you requested. This is the file after it was run through formware.
[CE3_HH Bust_fixed.3mf.zip](https://github.com/Ultimaker/Cura/files/8639913/CE3_HH.Bust_fixed.3mf.zip)
.

**Comment by GregValiant** [comment]
Slicing with your settings in 5.0 I'm not seeing those errors but my video system is in compatibility mode.
Here is a project file with most of the model blocked out.  Try slicing and printing this as a test (it's just a 10 minute print).

[GV_HH Bust.zip](https://github.com/Ultimaker/Cura/files/8641778/GV_HH.Bust.zip)

**Comment by GT-Geo** [comment]
Thanks again Greg, slicing it looks to have similar errors. The printed part also is full of those errors. 
<img width="1725" alt="Screen Shot 2022-05-06 at 1 46 51 PM" src="https://user-images.githubusercontent.com/104281051/167192091-75e4211c-1152-4c71-bfb6-1d9271810ae9.png">
![fr_9414_size880](https://user-images.githubusercontent.com/104281051/167192860-eb5e2132-3064-4e10-a7f7-0fc8f577cb91.jpg)

**Comment by GregValiant** [comment]
It's tough to get a good picture.
What I think we're seeing is that the .4 nozzle is too big for the text.  When details fall below what Cura deems to be possible (given the hardware and settings) it "disappears" those features.
I printed that short version on my nicely tuned Ender 3 Pro.
On top is your model.  You can see that "Harry" isn't too bad but that "Houdini" isn't too good.  It's missing pieces and because the letters are so close together there is some stringing making it even fuzzier.
On the bottom is the model scaled at 150%.  Same .4 nozzle but now the features that were gone from the small one are wide enough to print.  (maybe could have picked a better color filament but the silky silver was in the printer).
![DSCN2858](https://user-images.githubusercontent.com/64202104/167216264-d7ac0b5f-76f3-4aa9-b288-9a2f7ae506c0.JPG)

Somebody from the Cura team will look at this when they get a chance.  What you could do in Cura is to go to a .2 nozzle and .2 line width and look at the preview of your model and see if those missing areas show up.

**Comment by GT-Geo** [comment]
This was sliced with Cura 4.13.1 and printed super clean with a .4 nozzle.  I can't this result with Cura 5.

![fr_9416_size880](https://user-images.githubusercontent.com/104281051/167220448-e76f5a8b-5cf6-430a-bd2d-6b62a36061c7.jpg)

**Comment by GregValiant** [comment]
Did you try saving it as a project in 4.13 and opening the project in 5.0?  If it's one of the new wall settings in 5.0 that is out of adjustment then the project idea won't fix it because it won't affect it.
I did notice that in the 5.0 project you didn't have Print Thin Walls turned on.  Was it on in 4.13?

**Comment by GT-Geo** [comment]
Yes I just tried that, I saved it as a project in 4.13.1 then opened it in 5.0. It shows the same errors. 

Print Thin Walls is the same for both versions, not checked.

**Comment by MariskaMaas** [comment]
Hey @GT-Geo, thanks for reporting your issue.

@GregValiant is right, thanks Greg, Cura 5 handles walls in a different ways. Especially details like your letters are defined in a different way than in Cura 4. If you want to learn more about the changed settings. you can find it here https://support.ultimaker.com/hc/en-us/articles/4792077687068-What-new-print-settings-are-introduced-to-Cura-5-

I believe most of the trouble you have been experiencing come from the white zseam/retraction parts in your print preview. They dissapear for me if I reduce the Minimum Wall Line Width to 0.24 mm. This will be smaller than your nozzle, but it's still in the acceptable range so you might achieve the quality you are looking for. 

Can you try to change the Minimum Wall Line Width and let me know if this works for you? 
If not we can have a look at some of the other settings that can be changed to help you. 

Good luck!

![image](https://user-images.githubusercontent.com/40423138/170066406-62a3e9dd-725b-42f4-acc1-4f1f855053ca.png)
